android-core/src/androidTest/kotlin/com.mparticle/identity/IdentityApiTest.kt

```diff
@@ -111,26 +113,27 @@ class IdentityApiTest : BaseCleanStartedEachTest() {
 
         // test that change actually took place
         result.addSuccessListener { identityApiResult ->
-            Assert.assertEquals(identityApiResult.user.id, mpid1)
-            Assert.assertEquals(identityApiResult.previousUser!!.id, mStartingMpid.toLong())
+            Assert.assertEquals(identityApiResult.user.id, mStartingMpid)
+            // After Adding Identity caching, it uses previous response
+            // Assert.assertEquals(identityApiResult.previousUser!!.id, mStartingMpid.toLong())
```

Comment:
If we don't need these asserts, can we remove hem instead of commenting them out? Same for the other 2 in this file.